### DISCUSSION TITLE
Remote Script Execution Documentation

### DISCUSSION THREAD

**Discussion by peterjungx** [h2oai/wave#1158] [Q&A] [answered]
The documentation states that "Multiple Wave scripts running on multiple devices can update the same Wave page." [here](https://wave.h2o.ai/docs/scripts). Yet I cannot find how this is achieved.

What is missing is how to configure the machines IP on which `waved` is running, when the script needs to be executed on another machine.

**Comment by lo5** [comment, marked as answer]
Set the `H2O_WAVE_ADDRESS`, `H2O_WAVE_ACCESS_KEY_ID` and `H2O_WAVE_ACCESS_KEY_SECRET` env vars while running the script.

e.g.
```
$ H2O_WAVE_ADDRESS=https://192.168.42.42:8000 \
    H2O_WAVE_ACCESS_KEY_ID=foo \
    H2O_WAVE_ACCESS_KEY_SECRET=bar \
    python my_script.py
```

See https://wave.h2o.ai/docs/configuration#h2o_wave_address
